railties/test/isolation/abstract_unit.rb

```diff
@@ -236,10 +236,88 @@ def engine(name)
       end
     end
 
-    def script(script)
-      Dir.chdir(app_path) do
-        `#{Gem.ruby} #{app_path}/bin/rails #{script}`
+    # Invoke a bin/rails command inside the app
+    #
+    # allow_failures:: true to return normally if the command exits with
+    #   a non-zero status. By default, this method will raise.
+    # stderr:: true to pass STDERR output straight to the "real" STDERR.
+    #   By default, the STDERR and STDOUT of the process will be
+    #   combined in the returned string.
+    # fork:: false to not use fork even when it's available. By default,
+    #   when possible, the command is executed in a fork of the current
+    #   process, avoiding the need to load core Rails libraries anew.
+    def rails(*args, allow_failure: false, stderr: false, fork: true)
+      args = args.flatten
+
+      command = "bin/rails #{Shellwords.join args}#{' 2>&1' unless stderr}"
+
+      # Don't fork if the environment has disabled it
+      fork = false if ENV["NO_FORK"]
```

Comment:
What about to rename this var (or refactor into method `fork?`) to avoid having `fork` method (Kernel `fork`) and `fork` variable at the same context?

Reply:
If it was purely internal I would've just named it differently.. but I felt it was more important to keep the best name for the kwarg. Hopefully we can kill the argument (https://github.com/rails/rails/pull/30520#issuecomment-326949204), then it won't matter.